modules/electrophysiology_browser/jsx/react-series-data-viewer/README.md

```diff
@@ -32,24 +31,52 @@ The EEG Browser visualization component adds support for some useful visual help
+ - The **Timeline Range View** (2) can be used to change the boundaries of the viewed timeline.
+   - *[<] / [>]*: These arrows translate the interval bounds backwards or forwards, respectively, by 1 second.
+   - *[<<] / [>>]*: These arrows translate the interval bounds backwards or forwards, respectively, by the value of the interval.
+   - *Text fields*: The text fields can be edited to manually set the interval.
+   - *Sliders*: The sliders can be dragged as an alternative way to set the interval range.
+ - The **Amplitude** and **Filter** tools and the **Show/Hide Overflow** button (3) can be used to increase/reduce the amplitude scale, apply high or low-pass filters, or toggle the visibility of signal spillage, respectively.
+ - The **Channel Navigation** (4) toolbar can be used to navigate through the viewed channels.
+   - *Dropdown*: This dropdown allow to change the number of displayed channels. Currently supported values are: 4, 8 16, 32 or 64 visible channels.
+   - *Text field*: This can be used to manually set the starting index of the displayed channels.
+   - *[<] / [>]*: These arrows translate the visible channel range backwards or forwards, respectively, by 1 channel.
+   - *[<<] / [>>]*: These arrows translate the visible channel range backwards or forwards, respectively, by the number of displayed channel.
+ - The **Channel Adjustment** (5) buttons can be used to vertically adjust the position of the signals.
+   - *DC/NO Offset*: This button toggles the subtraction of DC offset from the signals, used to center them with respect to their assigned row.
+   - *Stack/Spread*: This button toggles the channels from being in their assigned row to being all stacked on the same row. [[Stacked View Demo](#stacked-view)]
+ - The **Event Panel** (6) can be used to display information about the events when event data is available.
+   - If the selected timeline range contains more than 500 events, a message inside the panel indicates the user to reduce the boundaries of the timeline in order to display the event data.
 <br/><br/>
-<p align="center">
-  <img width="650" src="./../../images/signal-values.png"/><br/>
-  <sub>Signal Viewer with signal values and event data displayed.</sub>
+<p style="text-align: center;">
+  <img width src="./../../images/signal-values.png"/><br/>
+  <sub>Signal Viewer with hovered signal value and event data displayed.</sub>
 </p>
 
+
+### Stacked View
+Hovering channel names while in 'stacked' or 'spread (default)' view will thicken the respective signal(s). While in stacked view, a feature called "Isolate" becomes available. [[Isolate Mode Demo](#isolate-mode)]
+<br/><br/>![Stacked View](./../../images/signal-stacked.png) <br/>
+
+
+### Isolate Mode
+Hovering channel names while in 'isolate' mode will make that signal the only visible signal on the plot.
+<br/><br/>![Isolate Mode](./../../images/signal-isolated.png) <br/>
```

Comment:
I don't think these images should be here. The sentences make sense without them and I don't think they add enough value to justify the cost of adding binary files to our github repo.

Reply:
Hi @driusan  - I have an updated commit ready to push, but just for context, these images were added to be consistent with @laemtl's documentation, and I think, requested originally by Leigh.
@laemtl should we move these elsewhere?

Reply:
I don't know if we have the option to host the files on a server to avoid committing the images. Documentation wise, I think those images were a good addition to document the module's new features.

Reply:
@driusan This PR updates a README that was already containing images. Maybe we can discuss the images issue later to not hold this PR?

Reply:
No, once images are committed they're in the repo forever and need to be downloaded with every clone.